### ISSUE TITLE
laravel 5.3 - issues

### ISSUE THREAD

**Issue by paulcocis** [Zizaco/entrust#709]
route middleware and "routeNeedsRole" method not working. 

`BadMethodCallException in Macroable.php line 74:
Method filter does not exist.`

**Comment by asadi20** [comment]
I assume that entrust does not support laravel 5.3 now,
you can use laratrust

**Comment by kevosomi** [comment]
@paulcocis did you get a solution? facing the same issue here.

**Comment by oLDu67** [comment]
+1

**Comment by alihesari** [comment]
@paulcocis @kevosomi I have the same problem. what's the solution?

**Comment by Zmingg** [comment]
Yes,Route::filter() was removed. I use Middleware take the place of Route filters.

**Comment by smartman** [comment]
This happens when upgrading from Laravel 5.1 to 5.2 too
